src/Aquifer.JsEngine/Aquifer.JsEngine.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="JavaScriptEngineSwitcher.V8" Version="3.24.2" />
+        <PackageReference Include="Microsoft.ClearScript.Complete" Version="7.4.5" />
```

Comment:
This includes Windows, Linux, and MacOS nuget packages (as well as multiple CPU variants) so that all dev workstations and the build server (Linux) can properly load JS files.